lua/luasnip/loaders/from_vscode.lua

```diff
@@ -81,19 +81,25 @@ local function load_snippet_file(langs, snippet_set_path)
 								and parts.prefix
 							or { parts.prefix }
 						for _, prefix in ipairs(prefixes) do
-							table.insert(
-								lang_snips,
-								ls.parser.parse_snippet({
-									trig = prefix,
-									name = name,
-									dscr = parts.description or name,
-									wordTrig = true,
-								}, body)
-							)
+							local ls_conf = parts.luasnip or {}
+
+							local snip = ls.parser.parse_snippet({
+								trig = prefix,
+								name = name,
+								dscr = parts.description or name,
+								wordTrig = true,
+							}, body)
+
+							if ls_conf.autotrigger then
+								table.insert(auto_lang_snips, snip)
+							end
+
+							table.insert(lang_snips, snip)
```

Comment:
Double insert? Am I missing something?

Reply:
Should it go to just the auto list? I did that way the first time and ended by doing it like this as it's not clear to me which one should be, let me know what's your opinion and I will change it as you like.

Reply:
Just auto I think, having it in the normal one won't matter because it will auto expand before expand is pressed.
Not sure if autosnippets show up in cmp_luasnip, but that would be a separate issue.